bgpd/bgp_keepalives.c

```diff
@@ -180,11 +180,7 @@ void *bgp_keepalives_start(void *arg)
 	pthread_cond_init(peerhash_cond, &attrs);
 	pthread_condattr_destroy(&attrs);
 
-#ifdef GNU_LINUX
-	pthread_setname_np(fpt->thread, "bgpd_ka");
-#elif defined(OPEN_BSD)
-	pthread_set_name_np(fpt->thread, "bgpd_ka");
-#endif
+	frr_pthread_set_name(fpt, "bgpd_ka");
```

Comment:
By doing this you override the human readable name ("BGP Keepalives thread") given in `bgpd.c`, this is less readable than that. See end of review comment for a suggestion.